src/Transaction.proto

```diff
@@ -25,27 +25,38 @@ import "google/protobuf/timestamp.proto";
 
 package Catalyst.Protocol.Transaction;
 
+enum TransactionType {
+	NORMAL = 0;
+	CONFIDENTIAL = 1;
+}
+
 /**
  * Transaction
  *
- * Version: 1 for non-confidential transaction, 2 for confidential transaction
+ * TransactionType: 0 for non-confidential transaction, 1 for confidential transaction
```

Comment:
would it make sense to have another one for _SmartContract_ type of transaction

Reply:
Thought i'd follow the ether way of doing this. But this comment is valid imo.